### PR TITLE
Banner (Global Eyebrow)

### LINKED ISSUE #27: Add a page banner component
## Context

Our products often have a banner at the top of the page with a government website disclaimer, sometimes with [language links](https://www.consumerfinance.gov/).

image

### Pre-requisites

- [x] You have a machine with a Node.js v16 and yarn installed.

## Task

- [x] Clone, [set up](https://github.com/cfpb/design-stories#development), familiarize yourself with this repo and [Storybook](https://storybook.js.org/docs/react/get-started/introduction). Post a comment below if you run into any issues.
- [x] Create a `Banner.tsx` and `Banner.stories.tsx` file in the [components](https://github.com/cfpb/design-stories/tree/main/src/components) directory (use other filenames if preferred).
- [x] Create a React/Typescript version of cf.gov's [global eyebrow](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/jinja2/v1/includes/molecules/global-eyebrow.html) jinja2 template file.
- [x] The language links and phone number should be optional and configurable. It's probably best to keep them generic with a `links` prop instead of hard-coding anything.

Tips:

- The [react-uswds banner](https://trussworks.github.io/react-uswds/?path=/story/components-banner--custom-banner) is nearly identical to CFPB's so you should probably just copy/paste one (or all?) of [their banners](https://github.com/trussworks/react-uswds/tree/main/src/components/banner). Fun fact: The reason they look so similar is because USWDS copied the pattern from CFPB ~ 10 years ago. Hooray for coding in the open!
- It doesn't have to be a 1-to-1 duplicate of CFPB's HTML component. Just do an MVP version.

### PR DESCRIPTION
**Pull request by meissadia**
Closes #27 

## Changes

- Implements CFPB DS global eyebrow
- Adapts their .less styles (couldn't access .less variables or use functions such as unit())
- Adds stories, tests

## How to test this PR

1. git checkout banner && yarn 
2. yarn test
3. yarn start -> click around

## Screenshots
Screen Shot 2023-04-27 at 2 32 51 PM
 Screen Shot 2023-04-27 at 2 33 00 PM

### CODE REVIEW

File: src/components/Banner.less [hunk 1]
```diff
@@ -0,0 +1,117 @@
+@gray-40: #b4b5b6;
+@gray-5: #f7f8f9;
+
+.m-global-eyebrow {
+  font-size: 0.75em;
```
Comment (contolini) [inline src/components/Banner.less:5]: With the DS imported, you should be able to copy over the full [cf.gov eyebrow file](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/molecules/global-eyebrow.less) BUT we're using a newer version of Less so you'll have to add some parens around the first argument. e.g. Instead of

```less
  font-size: unit(12px / @base-font-size-px, em);
```

it'd be

```less
  font-size: unit((12px / @base-font-size-px), em);
```

File: src/components/header.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.wrapper {
+header .wrapper {
```
Comment (contolini) [inline src/components/header.css:1]: I think this file is unrelated to this PR?

File: src/components/Banner.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  } else {
+    eyebrowClasses.push('m-global-eyebrow__list');
+  }
+
+  return (
+    <div className={classnames(eyebrowClasses)} data-testid='eyebrow'>
+      <div className={classnames(wrapperClasses)}>
+        {taglineContent}
+
+        <div className='m-global-eyebrow_actions'>
```
Comment (contolini) [inline src/components/Banner.tsx:61]: This'll need to be a `<ul>` to play nice with its children `<li>`.

**Review by contolini — CHANGES_REQUESTED** [review]
👏 👏 👏 👏 👏

File: src/components/Banner.less [hunk 4]
```diff
@@ -0,0 +1,117 @@
+@gray-40: #b4b5b6;
+@gray-5: #f7f8f9;
```
[the remark this answers is not in the document]
Comment (meissadia) [reply src/components/Banner.less]: Thank you very much! I couldn't figure out what I was doing wrong here.

File: src/components/Banner.less (hunk 1 above)
Comment (meissadia) [reply src/components/Banner.less]: Perens!! Updated.  I didn't see these errors previously because I was importing the less variables incorrectly, so thanks for the guidance here.

**Review by contolini — APPROVED** [review]
